### PR TITLE
Update chat params

### PR DESCRIPTION
**Pull request by lusmoura**
## What this PR does

- Adds `temperature` as an option param to co.chat
- Adds `max_tokens` as an option param to co.chat
- Removes `preamble_override`
- Fixes the `user_name` param 

## How it was tested
- Unit tests

### CODE REVIEW

File: cohere/client.py [hunk 1]
```diff
@@ -154,8 +154,9 @@ def chat(self,
              return_chatlog: bool = False,
              return_prompt: bool = False,
              chatlog_override: List[Dict[str, str]] = None,
-             preamble_override: str = None,
-             username: str = None) -> Chat:
+             username: str = None,
```
Comment (sanderland) [inline cohere/client.py:157]: in #160 this is implemented as `user_name`

File: cohere/client.py [hunk 2]
```diff
@@ -154,8 +154,9 @@ def chat(self,
              return_chatlog: bool = False,
              return_prompt: bool = False,
              chatlog_override: List[Dict[str, str]] = None,
-             preamble_override: str = None,
```
Comment (lusmoura) [inline cohere/client.py:157]: [ref](https://github.com/cohere-ai/blobheart/commit/3f4c58c895fc9b75fccdb8b985d543a2783d5adc)

File: cohere/client.py (hunk 1 above)
Comment (lusmoura) [reply cohere/client.py:157]: should I change it here as well or is it better to just wait for #160 to be merged?

**Comment by sanderland** [comment]
Handled in #160

**Pull request closed without merging**